frontend/src/components/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -17,30 +21,68 @@ const SignUpForm: FC<Props> = ({ onSubmit }) => {
 
   return (
     <>
-      <h3>Sign Up</h3>
-      <form onSubmit={handleSubmit(onSubmit)}>
-        <p>
-          <Input
-            type="text"
-            label="Email"
-            placeholder="Enter your email"
-            name={getNameOf<UserSignUpRequestDto>('email')}
-            control={control}
-            errors={errors}
+      <div className={styles.main}>
+        <div className={styles.background}>
+          <div className={getValidClasses(styles.circle, styles.circleFirst)} />
+          <div className={getValidClasses(styles.circle, styles.circleSecond)}>
+            <img src={logo} className={styles.logo} />
+            <img src={authImage} />
+          </div>
```

Comment:
```suggestion
```
you need to add all similar things to the `frontend/src/components/auth/auth.tsx` (with styles)
and left only unique things in the SignIn and SignUp form